shimmingtoolbox/cli/maths.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+
+import click
+import os
+import nibabel as nib
+import numpy as np
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+AXES = ['0', '1', '2', '3', '4']
+
+
+@click.group(context_settings=CONTEXT_SETTINGS,
+             help="Perform mathematical operations on images.")
+def maths_cli():
+    pass
+
+
+@maths_cli.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--input', 'fname_input', type=click.Path(exists=True), required=True,
+              help="Input filename, supported extensions: .nii, .nii.gz")
+@click.option('--output', 'fname_output', type=click.Path(), default=os.path.join(os.curdir, 'mean.nii.gz'),
```

Comment:
a more intuitive default output would be to add "_mean" suffix to the input file